### PR TITLE
new fairphone 4 app and icon

### PR DESCRIPTION
**Pull request by aloxe**

### CODE REVIEW

File: other/appfilter.xml [hunk 1]
```diff
@@ -1558,7 +1558,6 @@
     <item component="ComponentInfo{com.samsung.android.calendar/com.samsung.android.app.calendar.activity.MainActivity}" drawable="calendar_31" />
     <item component="ComponentInfo{org.sufficientlysecure.localcalendar/org.sufficientlysecure.localcalendar.ui.MainActivity}" drawable="calendar_31" />
     <item component="ComponentInfo{org.lineageos.etar/com.android.calendar.AllInOneActivity}" drawable="calendar_31" />
-    <item component="ComponentInfo{com.google.android.calendar/com.android.calendar.AllInOneActivity}" drawable="calendar_31" />
```
Comment (aloxe) [inline other/appfilter.xml:1561]: this was a duplicate

**Comment by Quantenzitrone** [comment]
Icon is broken and has a "transform elements" in it, which the [contribution guide](https://github.com/Donnnno/Arcticons/blob/main/CONTRIBUTING.md) asks not have.
The transform elements attribute is also the reason for the R to be so weird.

**Comment by aloxe** [comment]
@Quantenzitrone These are the letters coming directly from the template font so I thought this was allowed in that case. I'll have a look at @Donnnno's fix to figure out.

**Comment by Donnnno** [comment]
Yeah, the lines were acting strange, also the word was too long for one line (so it looked cluttered), so changed that too.

**Pull request merged**